dashboard/test/models/user_test.rb

```diff
@@ -1563,6 +1563,18 @@ def track_progress(user_id, script_level, result, pairings: nil)
     assert_equal [], teacher.reload.permissions
   end
 
+  test 'assign_course_as_facilitator assigns course to facilitator' do
+    facilitator = create :facilitator
+    facilitator.course_as_facilitator = Pd::Workshop::COURSES.last
+    assert facilitator.courses_as_facilitator.where(course: Pd::Workshop::COURSES.last).any?
+  end
+
+  test 'delete_course_as_facilitator removes facilitator course' do
+    facilitator = create :facilitator
+    facilitator.delete_course_as_facilitator Pd::Workshop::COURSES.first
+    refute facilitator.courses_as_facilitator.where(course: Pd::Workshop::COURSES.first).any?
+  end
```

Comment:
See above. `assert_create` wasn't failing, but the return value wasn't usable. This is a different issue. In what way is it failing?

As to creating a `Pd::CourseFacilitator` entry in the facilitator factory, I think that should be a [trait](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#traits) and/or a [transient attribute](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#transient-attributes), but not the default behavior since it's not needed in most cases.